mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/ArenaAdapterService.kt

```diff
@@ -60,6 +65,17 @@ class ArenaAdapterService(
         return query { gjennomforing }
     }
 
+    private fun lookForExistingAvtale(tiltaksgjennomforing: ArenaTiltaksgjennomforingDbo): UUID? {
+        val tiltakstype = tiltakstyper.get(tiltaksgjennomforing.tiltakstypeId)
+            ?: throw IllegalStateException("Ukjent tiltakstype id=${tiltaksgjennomforing.tiltakstypeId}")
+
+        return if (Tiltakskoder.isTiltakMedAvtalerFraMulighetsrommet(tiltakstype.arenaKode)) {
```

Comment:
Vi gjør også en tilsvarende sjekk på tiltakskoder i frontend for å avgjøre om vi kan koble en gjennomføring mot avtale.

Vi kunne evt. vurdert å sjekke om `avtale.opphav == MR_ADMIN_FLATE` i stedet, men tror ikke det blir noen reell forskjell i oppførsel.